app/screens/auth/WalletConnectionType.tsx

```diff
@@ -114,7 +114,7 @@ const WalletConnectionType = ({ history, location }: AuthRouterParams) => {
             <Row>
               <Icon />
               <RowTitle>WALLET + NODE</RowTitle>
-              <Tooltip width={100} text="WALLET + NODE" />
+              <Tooltip width={120} text="To send, receive and mine SMH" />
```

Comment:
Not sure it is more clear, but usually we're calling it "smeshing" (like setting up smeshing / instead of setting up mining), so let's be consistent.
```suggestion
              <Tooltip width={120} text="To send, receive and smesh coins" />
```